### PR TITLE
Adding latest tag for catalog index image

### PR DESCRIPTION
**Pull request by varad-ahirwadkar**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -310,10 +310,11 @@ catalog-build: opm ## Build a catalog image.
 	echo "FROM quay.io/operator-framework/opm:$(OPM_VERSION)-ppc64le" >> $(TMP_DIR).Dockerfile
 	cat catalog/Dockerfile_final_stage >> $(TMP_DIR).Dockerfile
     ## Building catalog image
-	$(CONTAINER_TOOL) build -f $(TMP_DIR).Dockerfile -t $(CATALOG_IMG) .
-	rm -rf $(TMP_DIR)
+	$(CONTAINER_TOOL) build -f $(TMP_DIR).Dockerfile -t $(CATALOG_IMG) -t $(IMAGE_TAG_BASE)-catalog:latest .
+	rm -rf $(TMP_DIR)*
```
Comment (mkumatag) [inline Makefile:314]: this `*` thing in the end is always risk, be little cautious with that.

File: Makefile [hunk 2]
```diff
@@ -310,10 +310,11 @@ catalog-build: opm ## Build a catalog image.
 	echo "FROM quay.io/operator-framework/opm:$(OPM_VERSION)-ppc64le" >> $(TMP_DIR).Dockerfile
 	cat catalog/Dockerfile_final_stage >> $(TMP_DIR).Dockerfile
     ## Building catalog image
-	$(CONTAINER_TOOL) build -f $(TMP_DIR).Dockerfile -t $(CATALOG_IMG) .
-	rm -rf $(TMP_DIR)
+	$(CONTAINER_TOOL) build -f $(TMP_DIR).Dockerfile -t $(CATALOG_IMG) -t $(IMAGE_TAG_BASE)-catalog:latest .
```
Comment (mkumatag) [inline Makefile:313]: can you create one more variable like `CATALOG_IMG_LATEST` instead?

File: Makefile [hunk 3]
```diff
@@ -310,10 +310,11 @@ catalog-build: opm ## Build a catalog image.
...
+	rm -rf $(TMP_DIR)*
 
 # Push the catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
+	$(MAKE) docker-push IMG=$(IMAGE_TAG_BASE)-catalog
```
Comment (mkumatag) [inline Makefile:319]: same above variable can be used here.

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (varad-ahirwadkar) [reply Makefile]: sure, updated to 
```
	rm -rf $(TMP_DIR)
	rm -rf $(TMP_DIR).Dockerfile
```

File: Makefile (hunk 2 above)
Comment (varad-ahirwadkar) [reply Makefile]: sure, created the var `CATALOG_IMG_LATEST`.

File: Makefile (hunk 3 above)
Comment (varad-ahirwadkar) [reply Makefile]: Done, added and used the variable

File: Makefile [hunk 4]
```diff
@@ -310,10 +311,12 @@ catalog-build: opm ## Build a catalog image.
 	echo "FROM quay.io/operator-framework/opm:$(OPM_VERSION)-ppc64le" >> $(TMP_DIR).Dockerfile
 	cat catalog/Dockerfile_final_stage >> $(TMP_DIR).Dockerfile
     ## Building catalog image
-	$(CONTAINER_TOOL) build -f $(TMP_DIR).Dockerfile -t $(CATALOG_IMG) .
+	$(CONTAINER_TOOL) build -f $(TMP_DIR).Dockerfile -t $(CATALOG_IMG) -t $(CATALOG_IMG_LATEST) .
 	rm -rf $(TMP_DIR)
+	rm -rf $(TMP_DIR).Dockerfile
```
Comment (mkumatag) [inline Makefile:316]: If you are removing the directory itself in the above command, why do you need to delete this files explicitly?

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: Makefile (hunk 4 above)
Comment (varad-ahirwadkar) [reply Makefile:316]: It creates both a temporary directory and a temporary Dockerfile in the current directory.

Comment (mkumatag) [reply Makefile:316]: oops, my bad.. 🤦🏽‍♂️

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**